### ISSUE TITLE
"[pylint] Unused argument 'kwargs' [W0613]" when using Signals in model

### ISSUE THREAD

**Issue by thinh9e** [PyCQA/pylint-django#218]
### My code:
```python
from django.dispatch import receiver
from django.db.models.signals import post_save

@receiver(post_save, sender=settings.AUTH_USER_MODEL)
def post_save_func(sender, instance, **kwargs):
    ...
    pass
```
I don't use `**kwargs` in my function, but `**kwargs` is a mandatory variable to using `post_save`. Therefor, I have a warning "[pylint] Unused argument 'kwargs' [W0613]". Can you fix to hide warning in this case?
Thanks!
### Output `$ pip freeze`:
```
astroid==2.1.0
beautifulsoup4==4.7.1
certifi==2018.11.29
chardet==3.0.4
Click==7.0
colorama==0.4.1
colorlover==0.2.1
dash==0.35.1
dash-core-components==0.42.1
dash-html-components==0.13.4
dash-renderer==0.16.1
decorator==4.3.0
Django==2.1.5
djangorestframework==3.9.0
Flask==1.0.2
Flask-Compress==1.4.0
idna==2.8
ipython-genutils==0.2.0
isort==4.3.4
itsdangerous==1.1.0
Jinja2==2.10
jsonschema==2.6.0
jupyter-core==4.4.0
lazy-object-proxy==1.3.1
lxml==4.3.0
MarkupSafe==1.1.0
mccabe==0.6.1
nbformat==4.4.0
numpy==1.15.4
pandas==0.23.4
pandas-datareader==0.7.0
Pillow==5.4.1
plotly==3.5.0
pylint==2.2.2
pylint-django==2.0.5
pylint-plugin-utils==0.4
python-dateutil==2.7.5
pytz==2018.9
requests==2.21.0
retrying==1.3.3
six==1.12.0
soupsieve==1.6.2
stripe==2.17.0
traitlets==4.3.2
urllib3==1.24.1
Werkzeug==0.14.1
wrapt==1.11.0
```

**Comment by Qu4tro** [comment]
As an alternative, you can safely use `**_kwargs`.

**Comment by JensTimmerman** [comment]
Hi, 
I'm having the same issue and I'm intrested in working on implementing a fix for this, are there any pointers on how to start working on this?

**Comment by atodorov** [comment]
@JensTimmerman see `ignore_unused_argument_warnings_for_request` in `augmentations/__init__.py` and how it is used. You first need to add a test scenario for this issue, then figure out which one of the core pylint checkers is triggering the warning (look into pylint's source code) and then come up with a solution, similar to the one we already have. (honestly I baffles me why the current one doesn't work). 

For a general reference about pylint plugins see https://www.youtube.com/watch?v=3CkSKUNMLJc, the interesting stuff related to checkers & augmentations is at the end, after the Q&A session.
